### PR TITLE
sequelize structure refactor

### PR DESCRIPTION
**Pull request by alanhala**
### Summary

- changed sequelize structure

### CODE REVIEW

File: config/db.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const config = require('../config').common.database;
+
+module.exports = {
+  development: {
+    username: 'onboarding-digital',
```
Comment (gonziesc) [inline config/db.js]: This should not be here

File: config/index.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ const config = {
     },
     session: {
       header_name: 'authorization',
-      secret: process.env.NODE_API_SESSION_SECRET
+      secret: 'asdasd'
```
Comment (gonziesc) [inline config/index.js]: lol

File: console.js [hunk 3]
```diff
@@ -22,11 +22,11 @@ const convertObjectFunctionsToAsync = serviceMethods => {
   return asyncServiceMethods;
 };
 
-orm.init().then(() => {
+Promise.resolve().then(() => {
```
Comment (gonziesc) [inline console.js]: There is no need of this, anyways i'll update this script after this pr is merged

File: config/db.js (hunk 1 above)
Comment (alanhala) [reply config/db.js]: lol

**Pull request merged**